src/planscape/planning/migrations/0022_remove_projectarea_origin_scenario_origin.py

```diff
@@ -0,0 +1,33 @@
+from django.db import migrations, models
+
+
+def set_origin(apps, schema_editor):
+    from planning.models import ScenarioOrigin
+
+    Scenario = apps.get_model("planning", "Scenario")
+    Scenario.objects.all().update(origin=ScenarioOrigin.OPTIMIZATION)
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("planning", "0021_planningarea_deleted_at_scenario_deleted_at_and_more"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="projectarea",
+            name="origin",
+        ),
+        migrations.AddField(
+            model_name="scenario",
+            name="origin",
+            field=models.CharField(
+                choices=[
+                    ("OPTIMIZATION", "Optimization"),
+                    ("USER_CREATED", "User Created"),
```

Comment:
This all looks good so far, but I might be confused -- I thought our intention for moving the origin field here was to differentiate between Scenarios that were created via the UI/configuration, and Scenarios that were uploaded.
It definitely makes sense that a Project Area would have these two values, but they are a little confused if they're on a scenario model.